drivers/scheduler/k8s/k8s.go

```diff
@@ -864,6 +864,42 @@ func (k *K8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.
 
 		logrus.Infof("[%v] Created Role Binding: %v", app.Key, snap.Name)
 		return snap, nil
+	} else if obj, ok := spec.(*rbacv1.ClusterRole); ok {
+		obj.Namespace = ns.Name
+		snap, err := k8sRbac.CreateClusterRole(obj)
```

Comment:
rename snap variable name as it doesn't related to cluster role object

Reply:
sounds good, I am also going to change other objects using the same variable name `snap`